jdi-light-parallel-examples/src/test/java/io/github/epam/parallel/Preconditions.java

```diff
@@ -0,0 +1,49 @@
+package io.github.epam.parallel;
+
+import com.epam.jdi.light.elements.composite.WebPage;
+import io.github.epam.entities.User;
+import io.qameta.allure.Step;
+
+import static io.github.epam.EpamGithubSite.header;
+import static io.github.epam.EpamGithubSite.homePage;
+import static io.github.epam.entities.Users.DEFAULT_USER;
+
+public class Preconditions {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'Preconditions' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5133577)